arch/arm/boot/dts/bcm2835-rpi-zero-w-rust.dts

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2017 Stefan Wahren <stefan.wahren@i2se.com>
+ */
```

Comment:
Perhaps add a comment here explaining that this is copied from from the version without `-rust` and highlighting the difference (`bcm2835-rng-rust`)?

Reply:
@wedsonaf the devicetrees are forked temporarily, until the `platdev` can bind to a devicetree `compatible` string. Which will be in PR 2/5.

In other words, these forked devicetrees will be "history" in the next PR. I'm sorry that I didn't indicate this anywhere !

In that light, do you still feel it's worth writing/reviewing the devicetree comments? If so I'll add them, no problem.

Reply:
Thanks for the clarification. No need for additional comments.